pkg/scheduler/apis/config/v1alpha1/conversion_test.go

```diff
@@ -191,3 +193,206 @@ func TestConfigToV1alpha1KubeSchedulerLeaderElectionConfiguration(t *testing.T)
+		},
+		Unreserve: &v1alpha1.PluginSet{
+			Enabled: []v1alpha1.Plugin{
+				{Name: "unreserve-plugin", Weight: &dummyWeight},
+			},
+			Disabled: []v1alpha1.Plugin{
+				{Name: "disabled-unreserve-plugin", Weight: &dummyWeight},
+			},
+		},
+	}
+	configPlugins := config.Plugins{
+		QueueSort: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "queuesort-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-queuesort-plugin", Weight: dummyWeight},
+			},
+		},
+		PreFilter: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "prefilter-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-prefilter-plugin", Weight: dummyWeight},
+			},
+		},
+		Filter: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "filter-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-filter-plugin", Weight: dummyWeight},
+			},
+		},
+		PreScore: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "postfilter-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-postfilter-plugin", Weight: dummyWeight},
+			},
+		},
+		Score: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "score-plugin", Weight: weight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-score-plugin", Weight: weight},
+			},
+		},
+		Reserve: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "reserve-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-reserve-plugin", Weight: dummyWeight},
+			},
+		},
+		Permit: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "permit-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-permit-plugin", Weight: dummyWeight},
+			},
+		},
+		PreBind: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "prebind-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-prebind-plugin", Weight: dummyWeight},
+			},
+		},
+		Bind: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "bind-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-bind-plugin", Weight: dummyWeight},
+			},
+		},
+		PostBind: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "postbind-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-postbind-plugin", Weight: dummyWeight},
+			},
+		},
+		Unreserve: &config.PluginSet{
+			Enabled: []config.Plugin{
+				{Name: "unreserve-plugin", Weight: dummyWeight},
+			},
+			Disabled: []config.Plugin{
+				{Name: "disabled-unreserve-plugin", Weight: dummyWeight},
+			},
+		},
+	}
+	convertedConfigPlugins := config.Plugins{}
+	convertedV1Alpha1Plugins := v1alpha1.Plugins{}
+	s := conversion.Scope(nil)
+	if err := Convert_v1alpha1_Plugins_To_config_Plugins(&v1alpha1Plugins, &convertedConfigPlugins, s); err != nil {
```

Comment:
instead of calling these functions directly, could you create an scheme, call `AddToScheme` and then call `Convert`?

Reply:
Fixed